uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -386,7 +397,7 @@ define([
         //if set indicates self leaverequest is either being managed or edited
         if (this.directiveOptions.leaveRequest) {
           //get a clone so self it is not the same reference as passed from callee
-          attributes = _.cloneDeep(this.directiveOptions.leaveRequest.attributes());
+          attributes = this.directiveOptions.leaveRequest.attributes();
```

Comment:
Also in the call we talked about using `angular.copy` as `cloneDeep` was not making a perfect copy, dont we need that?